src/app/shared/services/skin/skin.service.ts

```diff
@@ -1,48 +1,58 @@
 import { Injectable } from "@angular/core";
 import { BehaviorSubject } from "rxjs";
 import { LocalStorageService } from "src/app/shared/services/local-storage/local-storage.service";
-import { APP_CONSTANTS } from "src/app/data";
-
-const { APP_FIELDS } = APP_CONSTANTS;
-
-// TODO - Where should these live and be added to? Should ultimately come from sheets?
-// Could live at deployment level, e.g. app_constants.compatible_skins
-const SKINS = ["default", "modular"];
+import { APP_CONFIG } from "src/app/data";
+import { IAppSkin } from "data-models";
+import { arrayToHashmap } from "../../utils";
+import { AppConfigService } from "../app-config/app-config.service";
 
 @Injectable({
   providedIn: "root",
 })
 export class SkinService {
-  currentSkin$ = new BehaviorSubject<string>(null);
-  availableSkins = SKINS;
+  // A hashmap of all skins available to the current deployment
+  availableSkins: { [skinName: string]: IAppSkin };
+  activeSkin$ = new BehaviorSubject<IAppSkin | {}>({});
 
-  constructor(private localStorageService: LocalStorageService) {}
+  constructor(
+    private localStorageService: LocalStorageService,
+    private appConfigService: AppConfigService
+  ) {}
 
   init() {
-    const currentSkinName = this.getCurrentSkin();
-    if (currentSkinName) {
-      this.setSkin(currentSkinName);
-    } else {
-      this.setSkin("default");
-    }
+    const skinsConfig = this.appConfigService.APP_CONFIG.APP_SKINS;
+    this.availableSkins = arrayToHashmap([...skinsConfig.available, skinsConfig.default], "name");
+    // Retrieve the last active skin with default fallback
+    const activeSkinName = this.getActiveSkinName() ?? skinsConfig.default.name;
+    // Set active skin
+    this.setSkin(activeSkinName);
```

Comment:
I think this floats back to my on-again off-again question of order of operations. Should hte skin service be initialised before the appConfig service so that the expected skin is loaded immediately? I'm not sure myself, but can imagine there being some potential confusing behaviour if we don't get this right (e.g. scheduling notifications using default title/text from the wrong skin, which then aren't rescheduled, or just duplicating all calls to other init functions)